adoptopenjdk-api-v3-frontend/src/main/resources/JSON/platforms.json

```diff
@@ -250,6 +250,24 @@
       "checksumAutoCommandHint": " (the command must be run on a terminal in the same directory you download the binary file)",
       "checksumAutoCommand": "curl -O -L FILEHASHURL && shasum -a 256 -c FILEHASHNAME",
       "osDetectionString": "not-to-be-detected"
+    },
+    {
+      "officialName": "Linux riscv64",
+      "searchableName": "RISCV64_LINUX",
+      "attributes": {
+        "heapSize": "normal",
+        "os": "linux",
+        "architecture": "riscv64"
+      },
+      "logo": "riscv.png",
```

Comment:
Wasn't sure what this file does/should be

Reply:
I am not sure this is even used anymore, probably fine

Reply:
it is used in the images on the website when we display hero cards

Reply:
yeah it's unused